tests/api/test_groups.py

```diff
@@ -272,13 +280,13 @@ def test_GroupsMixin_link_concession_group_funding_source_expiry(
     mock_ClientProtocol_post_link_concession_group_funding_source, mocker
 ):
     client = GroupsMixin()
-    mocker.patch.object(client, "_format_concession_expiry", return_value="formatted concession expiry")
+    mocker.patch.object(client, "_format_expiry", return_value="formatted concession expiry")
 
     result = client.link_concession_group_funding_source("group-1234", "funding-source-1234", datetime.now())
 
     endpoint = client.concession_group_funding_source_endpoint("group-1234")
     mock_ClientProtocol_post_link_concession_group_funding_source.assert_called_once_with(
-        endpoint, {"id": "funding-source-1234", "concession_expiry": "formatted concession expiry"}, dict
+        endpoint, {"id": "funding-source-1234", "expiry": "formatted concession expiry"}, dict
```

Comment:
Same nit about `concession`

Reply:
Removed in https://github.com/cal-itp/littlepay/pull/45/commits/484cf53ab421a1254e28da48bb39364e141b8f8c